### DISCUSSION TITLE
When to use and not to use  "standardize=TRUE" when calculating VIMP or Error?

### DISCUSSION THREAD

**Discussion by gundalav** [kogalur/randomForestSRC#150] [Q&A]
Hi, 

Thank you for providing such a wonderful package. 

I'm currently performing _multivariate_ RF for finding variable importance and error rate, 
following [this great tutorial of yours](https://luminwin.github.io/randomForestSRC/articles/getstarted.html#specifying-the-type-of-predicted-value-variable-importance-and-error-rate-1).

When I carried out the error calculation with this commands, with and without standardization, 
I found the result is almost complete opposite:

```
> get.mv.error(mv.obj, standardize = FALSE)
    0.3mM       1mM       3mM      10mM      20mM 
0.1735656 0.3179477 0.4107174 0.5389480 1.0541274 

> get.mv.error(mv.obj, standardize = TRUE)
    0.3mM       1mM       3mM      10mM      20mM 
0.8099730 0.3788312 0.2233027 0.2694740 0.4760575 

```

For example `0.3mM` error rate is lowest _without_ but highest _with_ standardization.

When should we opt for `standardize = FALSE` in our calculation?
Also what's the reasonable limit for the error rate?
Any rule of thumb? 

Thanks and hope to hear from you again. 

Sincerely, 
G.V

**Comment by ishwaran** [comment]
When you have a continuous Y outcome, unstandardized performance and variable importance (VIMP) are given in terms of mean-squared error.  Therefore these values are completely dependent on the scale of Y -- in fact if you scale Y by a constant, these values are scaled by that constant squared.   Therefore you cannot compare performance and VIMP across data sets or different outcomes as it is meaningless and would simply favor outcomes that just happened to be on a scale that is larger.  In particular, this means `standardize = TRUE ` should be the preferred way to compare performance and VIMP in multivariate RF.
